frontend/src/metabase/notifications/EmailAttachmentPicker.jsx

```diff
@@ -316,6 +346,22 @@ export default class EmailAttachmentPicker extends Component {
                 ))}
               </ul>
             </div>
+            <Group justify="space-between" pt="1rem">
+              <Group position="left" gap="0">
+                <Text fw="bold">{t`Send only attachments (no charts)`}</Text>
+                <Icon
+                  name="info"
+                  className={cx(CS.textMedium, CS.ml1)}
```

Comment:
Global classes are deprecated, please use `c` and `ml` props from mantine instead